### PR TITLE
Fix scripts

### PR DESCRIPTION
**Pull request by kay0u**
Fix scripts

Should fix this:
#3 
#8 
#16 
#17 

TODO :
- [x] Install
- [x] Remove
- [x] Upgrade
- [x] Backup
- [x] Restore
- [x] Readme

All tested at home :-)

### CODE REVIEW

File: scripts/backup [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
-# Exit on command errors and treat unset variables as an error
-set -eu
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Source app helpers
+source ../settings/scripts/_common.sh
```
Comment (frju365) [inline scripts/backup:9]: ? source .common.sh is enough :)

Comment (kay0u) [reply scripts/backup:9]: This is inspired by this:

https://github.com/YunoHost/example_ynh/blob/master/scripts/backup#L9

Backup and restore scripts are not like other scripts and do not run in the same place.

Comment (frju365) [reply scripts/backup:9]: oki :) It's just me (I'm so tired :/). Well, I didn't even check what was failing. Do you want to take care of this app with me ?

Comment (kay0u) [reply scripts/backup:9]: Yeah if you want!

Comment (kay0u) [reply scripts/backup]: Maybe I can upgrade the Readme script to be like this one?
https://github.com/YunoHost/example_ynh/blob/master/README.md

**Comment by kay0u** [comment]
Maybe I can upgrade the Readme to be like this one?
https://github.com/YunoHost/example_ynh/blob/master/README.md

File: scripts/backup (hunk 1 above)
Comment (frju365) [reply scripts/backup:9]: ok. I added you as admin of this repo. Better to be two than alone (I see so much app which are abandoned by their maintainer and taken again 1 year later).

**Comment by frju365** [comment]
yes if you want :) It could be nice

**Comment by kay0u** [comment]
Done!

**Pull request merged**